push_notification/subscriber/services/subscriber_service.py

```diff
@@ -0,0 +1,48 @@
+from subscriber.models.subscriber import Subscriber
+
+
+class SubscriberService(object):
+
+    def __init__(self, subscriber, serializer):
+        self.subscriber = subscriber
+        self.serializer = serializer
+
+    def get_subscriber(self, pk=None):
+        "Return subscriber details by given id"
+        if pk is not None:
+            queryset = self.subscriber.objects.get(id=pk)
+            response = self.serializer(queryset, many=False)
+            return response.data
+        return None
+
+    def get_all_subscriber(self):
+        """Return all subscriber registered"""
```

Comment:
Since the class name already says `SubscriberService` , we can remove `subscriber` from all method names as we already know which resource this service is trying to query